src/telemetry/metrics/mod.rs

```diff
@@ -136,9 +136,14 @@ struct Stamped<T> {
 /// is a Hyper service which can be used to create the server for the
 /// scrape endpoint, while the `Record` side can receive updates to the
 /// metrics by calling `record_event`.
```

Comment:
I think this comment should be updated to add that it also returns the `TlsConfigReload`?

Reply:
This is all going to change in the near future. I'll probably remove the comment entirely

Reply:
the return type is no longer changed, so no change is necessary yet